.github/workflows/weekly-version-bump.yml

```diff
@@ -0,0 +1,73 @@
+name: Bump versions of all DJ Services and Clients.
```

Comment:
Do we actually want a weekly auto-release? Or is it better off as a manual thing? Then we can be more intentional about when we release

Reply:
For the minor/major version we would still run this workflow manually, but the weekly automation would only bump the patch version. I think this should be automated.

Reply:
Ok sounds good 👍

Reply:
Instead of the patch version, why don't we publish [release candidates](https://peps.python.org/pep-0440/#pre-releases) with the `rc` tag in the version? That way these weekly automated pushes don't get picked up automatically everywhere the client isn't pinned. (pip will ignore `rc` versions unless specifically requested in the install command)

Reply:
Will pip also ignore the `alpha` and `beta` version tags? Because we are now using `alpha` everywhere. I don't think we should make this too complex at this stage of the project.

Reply:
Thinking more about it... once we get these versions in sync manually. What's wrong with trying to keep them in-sync... and yes, the weekly-auto process would only do alpha/beta/rc. And when we want to bump a big version we would run this workflow with a different argument (minor, major. etc). I still think that keeping them all in sync would be easy and great way to lower our brain cycles. What am I missing?

Reply:
Yeah, that's a good point. If we're going to do auto-releases, then it will be easier if we just keep all the versions aligned, and if we do a few extra releases that have nothing in them, that's fine

Reply:
Yeah I'm ok with that too. And I heard a great point the other day that we can always just include a simple argument to skip releases of certain components, for example if we're doing a release of everything from 0.0.1 to 0.0.2 but nothing has changed in the client, we could always add a flag to skip the client. Doesn't seem like an issue that the client version would go from 0.0.1 to 0.0.3 and it would allow everything to stay in sync.